packages/lavapack/src/builder-runtime.js

```diff
@@ -18,15 +18,20 @@ async function buildRuntimeCJS () {
 }
 
 async function buildRuntimeES (opts) {
+  const includeSnow = Boolean(opts.includeSnow)
   const runtimeTemplate = await readFile(pathJoin(__dirname, 'runtime-template.js'), 'utf8')
   let output = runtimeTemplate
+  if (includeSnow) {
+    output = getSnowSrc() + '\n\n' + output
+  }
   // inline kernel
   const kernelCode = generateKernel(opts)
   output = stringReplace(output, '__createKernel__', kernelCode)
   // inline reportStatsHook
   const statsCode = `(${makeInitStatsHook})({ onStatsReady })`
   output = stringReplace(output, '__reportStatsHook__', statsCode)
   output = markAsGenerated(output, 'runtime-template.js')
+  output = stringReplace(output, '__lavamoatBuilderOptions__', JSON.stringify({includeSnow}))
```

Comment:
Why is this needed at runtime? If SNOW is enabled, should SNOW source not just run on its own, without needing extra config?